EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
+            return $tokenUser->getAPIUser()->id;
+        }
+
+        if (method_exists($tokenUser, 'getId')) {
+            return $tokenUser->getId();
+        }
```

Comment:
It's not an eZ Publish interface. ( that's why I try with method_exist).
In my case it's an FOSUser interface as FOS is my external provider but it could be something else depending on the external provider.
=> that's why I try to know if we can get an "id" with method_exist.

?